lang/python/python-psycopg2/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-psycopg2
+PKG_VERSION:=2.8.6
+PKG_RELEASE:=2
+
+PYPI_NAME:=psycopg2
+PKG_HASH:=fb23f6c71107c37fd667cb4ea363ddeb936b348bbd6449278eb92c189699f543
+
+PKG_MAINTAINER:=Dmitry Trefilov <the-alien@live.ru>
```

Comment:
Agree with @neheb, should either assign to yourself or leave as empty.

Reply:
Assigned it to myself